src/test/regress/expected/multi_cluster_management.out

```diff
@@ -221,6 +221,20 @@ SELECT run_command_on_workers('GRANT ALL ON pg_dist_local_group TO node_metadata
  (localhost,57638,t,GRANT)
 (2 rows)
 
+SELECT run_command_on_workers('GRANT ALL ON ALL TABLES IN SCHEMA citus TO node_metadata_user');
```

Comment:
minor: we mostly aim to give as least grant as possible. I think for schema we should grant `USAGE` not `ALL`

Reply:
Ping on this, isn't this possible?

Reply:
as we insert into pg_dist_object this is not supported